MekHQ/src/mekhq/campaign/personnel/Person.java

```diff
@@ -967,15 +970,21 @@ public void procreate() {
 						tCal.add(GregorianCalendar.DAY_OF_YEAR, 40*7);
 						setDueDate(tCal);
 						campaign.addReport(getHyperlinkedName()+" has conceived");
+						if (campaign.getCampaignOptions().logConception()) {
+						    addLogEntry(campaign.getDate(), "Has conceived");
+				        }
 					}
 				} else if (getSpouse() != null) {
 					if (getSpouse().isActive() && !getSpouse().isDeployed() && getSpouse().getAge(campaign.getCalendar()) > 13) {
 						// 0.05% chance that this procreation attempt will create a child
-						if (Compute.randomInt(10000) < 2) {
+						if (Compute.randomInt(10000) < 200) {
```

Comment:
This change is not okay in this PR. You're changing the chance of procreation. It also doesn't match the chance mentioned in the comment, just like the changed code does.